decred/decred/dcr/rpc.py

```diff
@@ -6,12 +6,38 @@
 import base64
 import ssl
 
+import types
 from decred.util import tinyhttp
```

Comment:
Ugh. I didn't notice this before. Empty line should be after `types`, not before. 

@teknico, is it feasible to add something like [isort](https://github.com/timothycrosley/isort) to our workflow?

Reply:
@buck54321, I think so, I'll look into it.